packages/jsii/lib/assembler.ts

```diff
@@ -645,25 +704,38 @@ export class Assembler implements Emitter {
         if (currNs.name !== ns.name) {
           const currNsDecl = currNs.valueDeclaration ?? currNs.declarations[0];
           const nsDecl = ns.valueDeclaration ?? ns.declarations[0];
+
+          const refs = [
+            [currNs.name, currNsDecl] as const,
+            [ns.name, nsDecl] as const,
+          ].sort(([l], [r]) => l.localeCompare(r));
```

Comment:
A comment here would be nice. Was this just to ensure the names are always in the same order, or something more subtle?